docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -254,6 +255,17 @@ Metriport makes it easy to share this data back with the networks - you can:
 - upload FHIR data using our [Create Patient's Consolidated Data](/medical-api/api-reference/fhir/create-patient-consolidated) endpoint.
 - upload binary documents using the [Upload Document](/medical-api/api-reference/document/post-upload-url) endpoint.
 
+## 10. Invite your team 🤝
+
+You can invite your team members to signup at https://dash.metriport.com/ and send us a list of the emails to be associated with your main account.
```

Comment:
perhaps clarify where they can send us the list of emails

Reply:
That's a good point, but currently the channels used for communication w/ our customers are pretty clear, they use either Slack or email.

I'd leave for a new PR once we learn more how this process will work.